tests/End2End/App/json_serializer.yaml

```diff
@@ -10,13 +10,13 @@ services:
       - [
           '@Symfony\Component\Serializer\Normalizer\ArrayDenormalizer',
           '@Symfony\Component\Serializer\Normalizer\DateTimeNormalizer',
-          '@Facile\MongoDbMessenger\Tests\End2End\App\SuppressDeprecationNormalizer'
+          '@Symfony\Component\Serializer\Normalizer\ObjectNormalizer'
         ]
       - ['@Symfony\Component\Serializer\Encoder\JsonEncoder']
 
   Symfony\Component\Serializer\Normalizer\ArrayDenormalizer: ~
   Symfony\Component\Serializer\Normalizer\DateTimeNormalizer: ~
-  Facile\MongoDbMessenger\Tests\End2End\App\SuppressDeprecationNormalizer: ~
+  Symfony\Component\Serializer\Normalizer\ObjectNormalizer: ~
   Symfony\Component\Serializer\Encoder\JsonEncoder: ~
```

Comment:
Since we dropped the specific normalizer, maybe we can drop the whole service definition? Or it's just temporary to see if we can solve the deprecations?